client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -40,6 +41,10 @@ const hosting: Flow = {
 	},
 	useStepNavigation( _currentStepSlug, navigate ) {
 		const { setPlanCartItem } = useDispatch( ONBOARD_STORE );
+		const getPlanCartItem = useSelect(
+			( select ) => ( select( ONBOARD_STORE ) as OnboardSelect ).getPlanCartItem,
```

Comment:
Should `getPlanCartItem` be invoked here, and then pass in the step slug as a dependency? Or is it fine like this, and then invoke it when it's used (in the `processing` step).